src/fielddef.ts

```diff
@@ -112,14 +112,20 @@ export function cardinality(fieldDef: FieldDef, stats, filterNull = {}) {
     (stat.missing > 0 && filterNull[type] ? 1 : 0);
 }
 
-export function title(fieldDef: FieldDef) {
+export function title(fieldDef: FieldDef, showUnit: boolean) {
   if (isCount(fieldDef)) {
     return COUNT_DISPLAYNAME;
   }
+
+  var unit = '';
```

Comment:
Yeah.   Also, don't use `var` unless you have compelling reason.   Use `let` only when you need to mutate the variable.   Always try to use `const` when possible.  :)